src/lib/dataEntities/networks.ts

```diff
@@ -48,6 +48,7 @@ export type ParentChain =
   | (L2Network & Required<Pick<L2Network, 'partnerChainIDs'>>)
 
 export type Chain = L2Network
+export type OrbitChain = Chain & { isOrbit: boolean }
```

Comment:
Let's remove `Chain` and `ParentChain` entirely. Also, for an `OrbitChain` type, can we make `isOrbit: true` in the type? 
```suggestion
export type OrbitChain = L2Network & { isOrbit: true }
```